pingdom-checks/veteran-verification.ping

```diff
@@ -25,3 +26,29 @@ pingdom save-check \
   -a group=veteran-verification \
   -a userids_csv="$STATUSPAGE_SANDBOX_VETERAN_VERIFICATION" \
   -a integrationids_csv="$VETERAN_VERIFICATION_SLACK_ID,$PAGER_DUTY"
+
+#sandbox
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-veteran-verification-v2-health \
+  -a host=sandbox-api.va.gov \
+  -a url="/services/veteran_verification/v2/healthcheck/liveness" \
+  -a port="443" \
+  -a responsetime_threshold="30000" \
+  -a resolution="1" \
+  -a group=veteran-verification \
+  -a userids_csv="$NO_USERS" \
+  -a integrationids_csv="$VETERAN_VERIFICATION_SLACK_ID,$PAGER_DUTY"
+
+#sandbox
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-veteran-verification-v2-health \
```

Comment:
The 'name' field must be unique for each probe; otherwise an existing probe in Pingdom would be overwritten. For the readniness endpoint, use sandbox-veteran-verification-v2-**backend**-health

Reply:
O gotcha will make that change